src/main/java/fi/metatavu/metamind/server/keycloak/AuthenticationController.java

```diff
@@ -285,6 +331,7 @@ public UUID getCreateResponseId(Response response) {
       Keycloak keycloak = getAdminClient();
       RealmResource realm = keycloak.realm(REALM);
       ClientRepresentation client = getClient(keycloak);
+      ClientRepresentation apiClient = getClient(getApiClient());
```

Comment:
could we retieve api client with admin client (keycloak)? This way we would not need to create two resource consuming clients

Reply:
fixed